predicates/content/content_length_between.go

```diff
@@ -0,0 +1,47 @@
+package content
+
+import (
+	"github.com/zalando/skipper/predicates"
+	"github.com/zalando/skipper/routing"
+	"net/http"
+)
+
+type contentLengthBetweenSpec struct{}
+
+type contentLengthBetweenPredicate struct {
+	min int64
+	max int64
+}
+
+// NewContentLengthBetween creates a predicate specification,
+// whose instances match content length header value in range from min to max inclusively.
+// example: ContentLengthBetween(0, 5000)
+func NewContentLengthBetween() routing.PredicateSpec { return &contentLengthBetweenSpec{} }
+
+func (*contentLengthBetweenSpec) Name() string {
+	return predicates.ContentLengthBetweenName
+}
+
+// Create a predicate instance that evaluates content length header value range
+func (*contentLengthBetweenSpec) Create(args []interface{}) (routing.Predicate, error) {
+	if len(args) != 2 {
+		return nil, predicates.ErrInvalidPredicateParameters
+	}
+	minLength, _ := args[0].(int64)
+	maxLength, _ := args[1].(int64)
```

Comment:
Please check conversion success and return `ErrInvalidPredicateParameters` accordingly to prevent e.g. passing strings as arguments.

Also Skipper/eskip native number type is `float64` so you'd need to do something like:
```go
minLength := 0
if x, ok := args[0].(float64); !ok || float64(int(x)) != x {
    return nil, predicates.ErrInvalidPredicateParameters
} else {
    minLength = x
}
```
(See related https://github.com/zalando/skipper/pull/1814)